ts/types/pagopa.ts

```diff
@@ -146,13 +147,19 @@ export type Transaction = t.TypeOf<typeof Transaction>;
 
 export const isCompletedTransaction = (tx: Transaction) => tx.idStatus === 3;
 
+const idStatusSuccessTransaction: ReadonlyArray<number> = [8, 9];
 /**
- * accountingStatus === 1 means the transaction has been
- * confirmed and the payment has been successfully
- * completed
+ * to determine if a transaction is successfully completed we have to consider 2 cases
+ * 1. payed /w CREDIT CARD: accountingStatus is not undefined AND accountingStatus === 1 means the transaction has been
+ * confirmed and the payment has been successfully completed
+ * 2.payed /w other methods: accountingStatus is undefined AND id_status = 8 (Confermato mod1) or id_status = 9 (Confermato mod2)
```

Comment:
```suggestion
 * 2.payed /w other methods: accountingStatus is undefined AND id_status = 8 (Confirmed mod1) or id_status = 9 (Confirmed mod2)
```

Reply:
It's better to not traslate a name of a specific domain

see description here https://www.pivotaltracker.com/n/projects/2048617/stories/173850410

Reply:
@Undermaken ok! at a glance I didn't realize it was a specific domain